SecurityService.OpenIdConnect.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -5,38 +5,158 @@
 namespace SecurityService.IntergrationTests.Common
 {
     using System.Data.SqlClient;
+    using System.Diagnostics;
+    using System.IO;
+    using System.Linq;
     using System.Net;
     using System.Net.Http;
     using System.Threading;
     using System.Threading.Tasks;
     using BoDi;
     using Client;
     using Ductus.FluentDocker.Builders;
+    using Ductus.FluentDocker.Common;
     using Ductus.FluentDocker.Model.Builders;
     using Ductus.FluentDocker.Services;
     using Ductus.FluentDocker.Services.Extensions;
     using OpenQA.Selenium;
     using OpenQA.Selenium.Chrome;
+    using Shared.Logger;
     using TechTalk.SpecFlow;
 
-    public class DockerHelper
+    public class DockerHelper : Shared.IntegrationTesting.DockerHelper
     {
-        public INetworkService TestNetwork;
+        private static void AddEntryToHostsFile(String ipaddress, String hostname)
+        {
+            if (FdOs.IsWindows())
+            {
+                using (StreamWriter w = File.AppendText(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.System), @"drivers\etc\hosts")))
+                {
+                    w.WriteLine($"{ipaddress} {hostname}");
+                }
+            }
+            else if (FdOs.IsLinux())
+            {
+                ExecuteBashCommand($"echo {ipaddress} {hostname} | sudo tee -a /etc/hosts");
+            }
+        }
 
-        public IContainerService SecurityServiceTestUIContainer;
+        static string ExecuteBashCommand(string command)
```

Comment:
Codacy found an issue: [Change return type to 'void'; not a single caller uses the returned value.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6144581)